code/controllers/master_controller.dm

```diff
@@ -95,10 +93,3 @@ datum/controller/game_controller/proc/setup_objects()
 	log_startup_progress("  Initialized [count] pipe networks in [stop_watch(watch)]s.")
 	
 	log_startup_progress("Finished object initializations in [stop_watch(overwatch)]s.")
-
-datum/controller/game_controller/proc/setup_starlight()

```

Comment:
Yea, I saw a comment that mentioned that. But there's no reason for the config NOT to load prior to starlight. I tested it: there's absolutely zero need for this. Bay, /vg/ and /tg/ don't have it either.